import utils from 'utils.js'

// error process method 1
// user execute try...catch
utils.foo(() => {
  try {

  } catch (e) {

  }
})
